num2words/lang_DE.py

```diff
@@ -70,9 +70,9 @@ def setup(self):
                      "ert": "erts",
                      "end": "ends",
                      "ion": "ions",
-                     "nen": "nens",
-                     "rde": "rdes",
-                     "rden": "rdens"}
+                     "nen": "ns",
+                     "rde": "rds",
```

Comment:
@fr-ser Thanks a lot for this fix! Could you add a test for this change too?

Reply:
Tests are updated as well. Should be good.